package.json

```diff
@@ -74,10 +75,11 @@
 	"dependencies": {
 		"@babel/runtime": "^7.22.15",
 		"dotenv": "^16.3.1",
-		"express": "^4.18.2",
+		"express": "^4.19.2",
 		"helmet": "^7.0.0",
 		"morgan": "^1.10.0",
 		"pg": "^8.11.3",
+		"serverless-http": "^3.2.0",
```

Comment:
Do we need this?

Reply:
do we need which line? I did not understand

Reply:
You added a dependency to `serverless-http`. Do wee need it?

Reply:
not anymore. I will get rid of it